CMakeLists.txt

```
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.21)
 project(imgui-xeno CXX)
 
 ## Error if not using switch toolchain file
-if (NOT SWITCH)
+if (NOT SWITCH AND NOT IMGUI_XENO_EXTERNAL_TOOLCHAIN)
   message(FATAL_ERROR "Not targeting switch, make sure to specify -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain.cmake")
```

I'd change the message to describe that a workaround is possible.

```suggestion
  message(FATAL_ERROR "Not targeting switch, make sure to specify -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain.cmake, or set IMGUI_XENO_EXTERNAL_TOOLCHAIN if another toolchain is already active")
```